torchvision/prototype/transforms/functional/_geometry.py

```diff
@@ -1012,10 +1237,13 @@ def perspective_image_tensor(
         coeffs=perspective_coeffs,
```

Comment:
Just saw this: is this the only place where we call the check function with `coeffs`? If yes, can't we just do the check "manually" here and keep the `coeffs` out of the affine parameters check?

Reply:
Let's do this on a follow up. The whole strategy of validation is fishy. For example we call methods such as `_affine_parse_args()` and then we call `_assert_grid_transform_inputs()`. This way we end up with lots of duplicate assertions, many of which are a bit useless or repetitive. Shall we review the assertion in these method separately to avoid blowing up further this PR?